pulsar-client-admin-api/src/main/java/org/apache/pulsar/client/admin/Topics.java

```diff
@@ -3693,4 +3694,47 @@ void setSubscriptionTypesEnabled(String topic,
      * @return a map of replicated subscription status on a topic
      */
     CompletableFuture<Map<String, Boolean>> getReplicatedSubscriptionStatusAsync(String topic, String subName);
+
+    /**
+     * Get the strategy used to check the a new schema provided by a producer is compatible with the current schema
+     * before it is installed.
+     *
+     * @param topic The topic in whose policy we are interested
+     * @return the strategy used to check compatibility
+     * @throws PulsarAdminException
+     *             Unexpected error
+     */
+    SchemaCompatibilityStrategy getSchemaCompatibilityStrategy(String topic) throws PulsarAdminException;
```

Comment:
We don't need add topic policies commands to `admin.topics()` any more, they are deprecated.